core/src/main/scala/kafka/controller/TopicDeletionManager.scala

```diff
@@ -267,6 +269,12 @@ class TopicDeletionManager(controller: KafkaController,
     controllerContext.removeTopic(topic)
   }
 
+  private def completePartitionReassignmentForDeletedTopic(topic: String): Unit = {
```

Comment:
This sounds more like `abortPartitionReassignmentForTopic`

Reply:
fixed